src/licenses.ts

```diff
@@ -161,10 +165,11 @@ const setGHLicenses = async (changes: Change[]): Promise<Change[]> => {
 
   return Promise.all(updatedChanges)
 }
+
 // Currently Dependency Graph licenses are truncated to 255 characters
 // This possibly makes them invalid spdx ids
 const truncatedDGLicense = (license: string): boolean =>
-  license.length === 255 && !isSPDXValid(license)
+  license.length === 255 && !spdx.isValid(license)
 
```

Comment:
🤔 this old codepath could be rethought - just a validity check (required anyway) on our inputs will catch these without the specific truncation checks. I didn't bother in this PR

Reply:
Yeah, that's a reasonable take. I'll also leave it for now.